app/jenkins-for-jira-ui/src/components/JenkinsSetup/JenkinsSetup.tsx

```diff
@@ -187,19 +179,25 @@ const IAmTheJenkinsAdmin = ({
 						</Button>
 					</div>
 				</li>
+				<li className={cx(orderedListItem, jenkinsSetupListItem)}>
+					Site name
+					<CopyButton
+						handleCopyToClipboard={handleCopyToClipboard}
+						copyRef={siteNameRef} testId="site-name" />
```

Comment:
This `testId` isn't very descriptive. I'd pick something that indicates it's a button that you can interact with like `site-name-copy-button`